JapaneseFilter/sections/specific.txt

```diff
@@ -35,6 +35,7 @@ news24.jp,michill.jp,fnn.jp,bengo4.com,zigsow.jp,navitime.com,cazual.shufu.co.jp
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+mobile.line.me###linemoMiniplan
```

Comment:
This rule doesn't work on subpages e.g. `https://mobile.line.me/guide/article/mnp-number.html`. Please change to `mobile.line.me##.lyBannerDiv`

Reply:
Fixed.